### ISSUE TITLE
LibreOffice applications do not run, no visible errors

### ISSUE THREAD

**Issue by ar-jan** [void-linux/void-packages#35200]
### System

* xuname:  
  *Void 5.15.16_1 x86_64 AuthenticAMD uptodate rrFFFFFFFFFFFF*
* package:  
  libreoffice-7.2.3.2_2

### Expected behavior
* Starting any of the LibreOffice applications (calc, writer, etc) should launch the application (launched from KDE start menu or KRunner).
* Alternatively, running e.g. `libreoffice --calc`  should start the application.

### Actual behavior
* The startup screen briefly appears, then nothing.
* Running from cli with `libreoffice --calc` does not produce any output either.

### Steps to reproduce the behavior
* Updated a previously working full *libreoffice* suite to the latest version.

I've tried to fully reinstall everything, but still see the same behavior:
`sudo xpbs-remove --recursive libreoffice`
`sudo xpbs-install libreoffice`

I hoped to see relevant errors in the logs, so I installed _socklog-void_ and enabled _socklog-unix_ and _nanoklogd_ services, but I do not see any errors or messages appear with `sudo svlogtail errors` or even `sudo svlogtail everything`.

**Comment by ar-jan** [comment]
Oh, so `sudo libreoffice --calc` does work!

**Comment by ar-jan** [comment]
It looks like somehow my session management was corrupted. I don't know how, I hadn't changed anything about my setup, but I saw this error `authpriv.err: login: pam_elogind(login:session): Failed to connect to system bus: No such file or directory`.

I had
* `elogind` installed and service enabled
* `dbus` installed and service enabled

But while `polkit` was installed, `polkitd` service was not (no longer?) enabled. I enabled `polkitd` and also configured KDE to start a new session (rather than restore an old one) and rebooted, and now LibreOffice works again.
